src/components/ProcessCreate/Steps/Confirm.tsx

```diff
@@ -343,14 +343,18 @@ const getCensus = async (env: EnvOptions, form: StepsFormValues, salt: string) =
       })
 
       const retryTime = 5000
-      const attempts = form.timeToCreateCensus / retryTime
-
       c3client.queueWait.retryTime = retryTime
-      // clamp attempts between 20 and 100
-      c3client.queueWait.attempts = Math.min(Math.max(Math.ceil(attempts), 20), 100)
```

Comment:
@selankon  I would not remove the above lines and I would just update the corresponding values in the case of gitcoin passport

Reply:
67825c3